apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerClient.Streaming.cs

```diff
@@ -28,7 +28,7 @@ public partial class SpannerClient
         /// <param name="request"></param>
         /// <param name="callSettings"></param>
         /// <returns></returns>
-        internal virtual AsyncServerStreamingCall<PartialResultSet> ExecuteSqlStream(ExecuteSqlRequest request, CallSettings callSettings = null)
+        public virtual AsyncServerStreamingCall<PartialResultSet> ExecuteSqlStream(ExecuteSqlRequest request, CallSettings callSettings = null)
```

Comment:
this class is modifying the partial autogenerated class.
some of this is to be replaced by work chris bacon is doing on streaming calls -- but not the api that returns a reliablestreamreader.

I actually think this is where these APIs need to be -- but I'm wondering if you noticed the class this is in.
What is the objection with modifying the partial class?